### ISSUE TITLE
Add "created" timestamp to tagged items for "most recently tagged" and "least recently tagged" filters

### ISSUE THREAD

**Issue by thenewguy** [jazzband/django-taggit#700]
It would be very helpful to have the creation timestamp for a tagged item.  It seems like this information would be generally useful and would allow adding `most_recent()` and `least_recent()` like `similar_objects()` and `most_common()`

**Comment by rtpg** [comment]
Yep that definitely makes sense (I tend to add created/updated timestamps on all of my  models as a general principle). For now it's easy to create a custom tag class to get this functionality though
